_tools/forcehost.md

```diff
@@ -0,0 +1,31 @@
+---
+parent: Tools
+layout: tool
+title: ForceHost
+year: '2022'
+platform:
+    - Web
+    - Faust
+availability: Available
+license: Open Source (GPL 3)
+haptic_category:
+    - Force Feedback
+hardware_abstraction: Bespoke (Force Host)
```

Comment:
```suggestion
hardware_abstraction: Bespoke (TorqueTuner device, optionally with ESP32 boards supporting audio I/O)
```

Reply:
Thank you! Since I've thought about using the details in parentheses for hardware abstraction in a new searching/filtering interface in the future, I'll accept these changes and then move the details to the additional information section. This should hopefully make sure it's all compatible when those changes get made.